.github/workflows/pull_request.yml

```diff
@@ -38,16 +36,10 @@ jobs:
           PYTHON_PYLINT_CONFIG_FILE: ${{ github.workspace }}/tests/.pylintrc
           VALIDATE_PYTHON_FLAKE8: false
           GITHUB_TOKEN: ${{ github.token }}
-      - name: Filter source files
```

Comment:
Don't filter the source file?

Reply:
Yeah. We have split sources and docs with different repo. So it's not necessary to do this filter.